tests/mocha/block_test.js

```diff
@@ -1030,13 +1021,6 @@ suite('Blocks', function() {
   });
   suite('Comments', function() {
     setup(function() {
-      Blockly.defineBlocksWithJsonArray([
-        {
-          "type": "empty_block",
-          "message0": "",
-          "args0": []
-        },
-      ]);
       this.eventSpy = sinon.spy(Blockly.Events, 'fire');
     });
     teardown(function() {
```

Comment:
Same here: the teardown should not remove the empty block

Reply:
Done